_posts/2018-04-28-hackillinois-2018-recap.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "Dreamt it. Built it. HackIllinois 2018, A Recap."
+date: 2018-04-28 00:00:00
+categories: recap
+tags: HackIllinois2018
+image: /assets/article_images/hackillinois_2018.jpg
+author: Justin Yang
+---
+
+This February, HackIllinois welcomed over a thousand students to the University of Illinois at Urbana-Champaign for an excitement-packed weekend to develop open-source projects.
+
+HackIllinois 2018 kicked off Friday afternoon, with students lining up for registration and to chat with sponsors at the company expo. After everyone filled up on dinner from Panda Express and Kohinoor, they heard opening remarks from the Opening Ceremony. Attendees then broke out into two panels based on their interests: Web & Data and Systems where open-source mentors spoke about their projects and the open-source community in general.
```

Comment:
This is picky, but I would do away with the comma and consider rephrasing the second part of the sentence

Reply:
I mean for the first sentence in this paragraph.